src/components/Cluster/NewCluster/CreateNodePoolsCluster.js

```diff
@@ -280,19 +280,23 @@ class CreateNodePoolsCluster extends Component {
       np => np.data
     );
 
+    let createPayload = {
```

Comment:
The variable declaration above is using `const`, so why not use `const` here? I've seen line 291 mutates the object but I think `const` would be the right variable declaration statement.

Reply:
If that's what you guys do in JS, I'll do it too ;-)

Reply:
After reading a but, I see that "`const` is a signal that the identifier won’t be reassigned.".

In our case, the identifier gets assigned to an object, and that object later gets modified. Is const the right thing then?

Reply:
You can modify properties in a `const` object, but can't reassign the whole object.  

`const` is our default when creating variables.

Reply:
ACK